src/components/nodes-table/index.vue

```diff
@@ -1,20 +1,16 @@
 <template>
   <v-data-table
-    :headers="activeHeaders"
-    :items="tableNodes"
+    :headers="managedNodes.getTableHeaders()"
+    :items="managedNodes.getFilteredItems()"
```

Comment:
I'm not 100% sure about this. Is this reacting to changes? AFAIK it's not a good practise to use a function that returns a value in this places

Reply:
Hmm, you may be right -  I'll have to think about it more deeply ...

Reply:
LMK

Reply:
I assume, just using the properties without getter and setter functions would solve that problem.
But then I wouldn't easily recognize changes (which is done in the `set*` functions) to automatically update the local storage.
Can Vuex help to solve this problem?
AFAIK getters and mutatations are splitted there as well.

Reply:
> I assume, just using the properties without getter and setter functions would solve that problem.
But then I wouldn't easily recognize changes (which is done in the set* functions) to automatically update the local storage.

I think that the correct way to do this is using the get/set methods in the class. You can use the set to detect changes

https://stackoverflow.com/questions/12827266/get-and-set-in-typescript

Reply:
> I think that the correct way to do this is using the get/set methods in the class. You can use the set to detect changes
> 
> https://stackoverflow.com/questions/12827266/get-and-set-in-typescript

LGTM, thanks for the hint! :-)

Reply:
Thanks to you :)

Reply:
It's done.